[TEST] SearchApiDocValueFieldsTests on master is failing.

```
[xUnit.net 00:01:38.5438681]     Tests.Search.Search.SearchApiDocValueFieldsTests.ReturnsExpectedIsValid [FAIL]
  X Tests.Search.Search.SearchApiDocValueFieldsTests.ReturnsExpectedIsValid [1s 195ms]
  Error Message:
   Tests.Framework.EndpointTests.ResponseAssertionException : Expected response.IsValid to be True because Invalid NEST response built from a unsuccessful (400) low level call on POST: /project/_search?pretty=true&error_trace=true&typed_keys=true
# Audit trail of this API call:
 - [1] BadResponse: Node: http://localhost:9200/ Took: 00:00:00.4281331
# OriginalException: Elasticsearch.Net.ElasticsearchClientException: Request failed to execute. Call: Status code 400 from: POST /project/_search?pretty=true&error_trace=true&typed_keys=true. ServerError: Type: search_phase_execution_exception Reason: "all shards failed" CausedBy: "Type: illegal_argument_exception Reason: "Invalid format: [weekyear]: Unknown pattern letter: r" CausedBy: "Type: illegal_argument_exception Reason: "Invalid format: [weekyear]: Unknown pattern letter: r" CausedBy: "Type: illegal_argument_exception Reason: "Unknown pattern letter: r""""
# Request:
{"docvalue_fields":["name",{"field":"lastActivity","format":"weekyear"}],"aggs":{"startDates":{"terms":{"field":"startedOn"}}},"from":10,"post_filter":{"term":{"state":{"value":"Stable"}}},"query":{"match_all":{}},"size":20}
# Response:
{
  "error" : {
    "root_cause" : [
      {
        "type" : "illegal_argument_exception",
        "reason" : "Invalid format: [weekyear]: Unknown pattern letter: r",
        "stack_trace" : "org.elasticsearch.ElasticsearchException$1: Invalid format: [weekyear]: Unknown pattern letter: r\r\n\tat org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:644)\r\n\tat org.elasticsearch.action.search.SearchPhaseExecutionException.guessRootCauses(SearchPhaseExecutionException.java:161)\r\n\tat org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:626)\r\n\tat org.elasticsearch.ElasticsearchException.generateFailureXContent(ElasticsearchException.java:572)\r\n\tat org.elasticsearch.rest.BytesRestResponse.build(BytesRestResponse.java:149)\r\n\tat org.elasticsearch.rest.BytesRestResponse.<init>(BytesRestResponse.java:110)\r\n\tat org.elasticsearch.rest.BytesRestResponse.<init>(BytesRestResponse.java:93)\r\n\tat org.elasticsearch.rest.action.RestActionListener.onFailure(RestActionListener.java:58)\r\n\tat org.elasticsearch.rest.action.RestCancellableNodeClient$1.onFailure(RestCancellableNodeClient.java:106)\r\n\tat org.elasticsearch.client.node.NodeClient.lambda$executeLocally$1(NodeClient.java:91)\r\n\tat org.elasticsearch.tasks.TaskManager$1.onFailure(TaskManager.java:184)\r\n\tat org.elasticsearch.action.search.AbstractSearchAsyncAction.raisePhaseFailure(AbstractSearchAsyncAction.java:576)\r\n\tat org.elasticsearch.action.search.AbstractSearchAsyncAction.onPhaseFailure(AbstractSearchAsyncAction.java:554)\r\n\tat org.elasticsearch.action.search.AbstractSearchAsyncAction.executeNextPhase(AbstractSearchAsyncAction.java:309)\r\n\tat org.elasticsearch.action.search.FetchSearchPhase.moveToNextPhase(FetchSearchPhase.java:230)\r\n\tat org.elasticsearch.action.search.FetchSearchPhase.lambda$innerRun$1(FetchSearchPhase.java:118)\r\n\tat org.elasticsearch.action.search.CountedCollector.countDown(CountedCollector.java:51)\r\n\tat org.elasticsearch.action.search.CountedCollector.onFailure(CountedCollector.java:70)\r\n\tat org.elasticsearch.action.search.FetchSearchPhase$2.onFailure(FetchSearchPhase.java:197)\r\n\tat org.elasticsearch.action.ActionListenerResponseHandler.handleException(ActionListenerResponseHandler.java:59)\r\n\tat org.elasticsearch.action.search.SearchTransportService$ConnectionCountingHandler.handleException(SearchTransportService.java:403)\r\n\tat org.elasticsearch.transport.TransportService$6.handleException(TransportService.java:579)\r\n\tat org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.handleException(TransportService.java:1112)\r\n\tat org.elasticsearch.transport.TransportService$DirectResponseChannel.processException(TransportService.java:1221)\r\n\tat org.elasticsearch.transport.TransportService$DirectResponseChannel.sendResponse(TransportService.java:1195)\r\n\tat org.elasticsearch.transport.TaskTransportChannel.sendResponse(TaskTransportChannel.java:61)\r\n\tat org.elasticsearch.transport.TransportChannel.sendErrorResponse(TransportChannel.java:56)\r\n\tat org.elasticsearch.action.support.ChannelActionListener.onFailure(ChannelActionListener.java:51)\r\n\tat org.elasticsearch.action.ActionRunnable.onFailure(ActionRunnable.java:88)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:39)\r\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:44)\r\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:706)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\r\n\tat java.base/java.lang.Thread.run(Thread.java:832)\r\nCaused by: java.lang.IllegalArgumentException: Invalid format: [weekyear]: Unknown pattern letter: r\r\n\tat org.elasticsearch.common.time.DateFormatters.forPattern(DateFormatters.java:1815)\r\n\tat org.elasticsearch.common.time.DateFormatter.forPattern(DateFormatter.java:145)\r\n\tat org.elasticsearch.index.mapper.DateFieldMapper$DateFieldType.docValueFormat(DateFieldMapper.java:436)\r\n\tat org.elasticsearch.search.fetch.subphase.FetchDocValuesPhase.hitsExecute(FetchDocValuesPhase.java:87)\r\n\tat org.elasticsearch.search.fetch.FetchPhase.execute(FetchPhase.java:184)\r\n\tat org.elasticsearch.search.SearchService.lambda$executeFetchPhase$4(SearchService.java:585)\r\n\tat org.elasticsearch.action.ActionRunnable.lambda$supply(ActionRunnable.java:58)\r\n\tat org.elasticsearch.action.ActionRunnable$2.doRun(ActionRunnable.java:73)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)\r\n\t... 6 more\r\nCaused by: java.lang.IllegalArgumentException: Unknown pattern letter: r\r\n\tat java.base/java.time.format.DateTimeFormatterBuilder.parsePattern(DateTimeFormatterBuilder.java:1813)\r\n\tat java.base/java.time.format.DateTimeFormatterBuilder.appendPattern(DateTimeFormatterBuilder.java:1710)\r\n\tat org.elasticsearch.common.time.DateFormatters.forPattern(DateFormatters.java:1811)\r\n\t... 14 more\r\n"
      }
    ],
    "type" : "search_phase_execution_exception",
    "reason" : "all shards failed",
    "phase" : "fetch",
    "grouped" : true,
    "failed_shards" : [
      {
        "shard" : 0,
        "index" : "project",
        "node" : "qIouw73JScmjqp8Y9eBJRw",
        "reason" : {
          "type" : "illegal_argument_exception",
          "reason" : "Invalid format: [weekyear]: Unknown pattern letter: r",
          "caused_by" : {
            "type" : "illegal_argument_exception",
            "reason" : "Unknown pattern letter: r",
            "stack_trace" : "java.lang.IllegalArgumentException: Unknown pattern letter: r\r\n\tat java.base/java.time.format.DateTimeFormatterBuilder.parsePattern(DateTimeFormatterBuilder.java:1813)\r\n\tat java.base/java.time.format.DateTimeFormatterBuilder.appendPattern(DateTimeFormatterBuilder.java:1710)\r\n\tat org.elasticsearch.common.time.DateFormatters.forPattern(DateFormatters.java:1811)\r\n\tat org.elasticsearch.common.time.DateFormatter.forPattern(DateFormatter.java:145)\r\n\tat org.elasticsearch.index.mapper.DateFieldMapper$DateFieldType.docValueFormat(DateFieldMapper.java:436)\r\n\tat org.elasticsearch.search.fetch.subphase.FetchDocValuesPhase.hitsExecute(FetchDocValuesPhase.java:87)\r\n\tat org.elasticsearch.search.fetch.FetchPhase.execute(FetchPhase.java:184)\r\n\tat org.elasticsearch.search.SearchService.lambda$executeFetchPhase$4(SearchService.java:585)\r\n\tat org.elasticsearch.action.ActionRunnable.lambda$supply(ActionRunnable.java:58)\r\n\tat org.elasticsearch.action.ActionRunnable$2.doRun(ActionRunnable.java:73)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)\r\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:44)\r\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:706)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\r\n\tat java.base/java.lang.Thread.run(Thread.java:832)\r\n"
          },
          "stack_trace" : "java.lang.IllegalArgumentException: Invalid format: [weekyear]: Unknown pattern letter: r\r\n\tat org.elasticsearch.common.time.DateFormatters.forPattern(DateFormatters.java:1815)\r\n\tat org.elasticsearch.common.time.DateFormatter.forPattern(DateFormatter.java:145)\r\n\tat org.elasticsearch.index.mapper.DateFieldMapper$DateFieldType.docValueFormat(DateFieldMapper.java:436)\r\n\tat org.elasticsearch.search.fetch.subphase.FetchDocValuesPhase.hitsExecute(FetchDocValuesPhase.java:87)\r\n\tat org.elasticsearch.search.fetch.FetchPhase.execute(FetchPhase.java:184)\r\n\tat org.elasticsearch.search.SearchService.lambda$executeFetchPhase$4(SearchService.java:585)\r\n\tat org.elasticsearch.action.ActionRunnable.lambda$supply(ActionRunnable.java:58)\r\n\tat org.elasticsearch.action.ActionRunnable$2.doRun(ActionRunnable.java:73)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)\r\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:44)\r\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:706)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\r\n\tat java.base/java.lang.Thread.run(Thread.java:832)\r\nCaused by: java.lang.IllegalArgumentException: Unknown pattern letter: r\r\n\tat java.base/java.time.format.DateTimeFormatterBuilder.parsePattern(DateTimeFormatterBuilder.java:1813)\r\n\tat java.base/java.time.format.DateTimeFormatterBuilder.appendPattern(DateTimeFormatterBuilder.java:1710)\r\n\tat org.elasticsearch.common.time.DateFormatters.forPattern(DateFormatters.java:1811)\r\n\t... 14 more\r\n"
        }
      }
    ],
    "caused_by" : {
      "type" : "illegal_argument_exception",
      "reason" : "Invalid format: [weekyear]: Unknown pattern letter: r",
      "caused_by" : {
        "type" : "illegal_argument_exception",
        "reason" : "Invalid format: [weekyear]: Unknown pattern letter: r",
        "caused_by" : {
          "type" : "illegal_argument_exception",
          "reason" : "Unknown pattern letter: r",
          "stack_trace" : "java.lang.IllegalArgumentException: Unknown pattern letter: r\r\n\tat java.base/java.time.format.DateTimeFormatterBuilder.parsePattern(DateTimeFormatterBuilder.java:1813)\r\n\tat java.base/java.time.format.DateTimeFormatterBuilder.appendPattern(DateTimeFormatterBuilder.java:1710)\r\n\tat org.elasticsearch.common.time.DateFormatters.forPattern(DateFormatters.java:1811)\r\n\tat org.elasticsearch.common.time.DateFormatter.forPattern(DateFormatter.java:145)\r\n\tat org.elasticsearch.index.mapper.DateFieldMapper$DateFieldType.docValueFormat(DateFieldMapper.java:436)\r\n\tat org.elasticsearch.search.fetch.subphase.FetchDocValuesPhase.hitsExecute(FetchDocValuesPhase.java:87)\r\n\tat org.elasticsearch.search.fetch.FetchPhase.execute(FetchPhase.java:184)\r\n\tat org.elasticsearch.search.SearchService.lambda$executeFetchPhase$4(SearchService.java:585)\r\n\tat org.elasticsearch.action.ActionRunnable.lambda$supply(ActionRunnable.java:58)\r\n\tat org.elasticsearch.action.ActionRunnable$2.doRun(ActionRunnable.java:73)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)\r\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:44)\r\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:706)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\r\n\tat java.base/java.lang.Thread.run(Thread.java:832)\r\n"
        },
        "stack_trace" : "java.lang.IllegalArgumentException: Invalid format: [weekyear]: Unknown pattern letter: r\r\n\tat org.elasticsearch.common.time.DateFormatters.forPattern(DateFormatters.java:1815)\r\n\tat org.elasticsearch.common.time.DateFormatter.forPattern(DateFormatter.java:145)\r\n\tat org.elasticsearch.index.mapper.DateFieldMapper$DateFieldType.docValueFormat(DateFieldMapper.java:436)\r\n\tat org.elasticsearch.search.fetch.subphase.FetchDocValuesPhase.hitsExecute(FetchDocValuesPhase.java:87)\r\n\tat org.elasticsearch.search.fetch.FetchPhase.execute(FetchPhase.java:184)\r\n\tat org.elasticsearch.search.SearchService.lambda$executeFetchPhase$4(SearchService.java:585)\r\n\tat org.elasticsearch.action.ActionRunnable.lambda$supply(ActionRunnable.java:58)\r\n\tat org.elasticsearch.action.ActionRunnable$2.doRun(ActionRunnable.java:73)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)\r\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:44)\r\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:706)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\r\n\tat java.base/java.lang.Thread.run(Thread.java:832)\r\nCaused by: java.lang.IllegalArgumentException: Unknown pattern letter: r\r\n\tat java.base/java.time.format.DateTimeFormatterBuilder.parsePattern(DateTimeFormatterBuilder.java:1813)\r\n\tat java.base/java.time.format.DateTimeFormatterBuilder.appendPattern(DateTimeFormatterBuilder.java:1710)\r\n\tat org.elasticsearch.common.time.DateFormatters.forPattern(DateFormatters.java:1811)\r\n\t... 14 more\r\n"
      },
      "stack_trace" : "org.elasticsearch.ElasticsearchException$1: Invalid format: [weekyear]: Unknown pattern letter: r\r\n\tat org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:644)\r\n\tat org.elasticsearch.action.search.AbstractSearchAsyncAction.executeNextPhase(AbstractSearchAsyncAction.java:307)\r\n\tat org.elasticsearch.action.search.FetchSearchPhase.moveToNextPhase(FetchSearchPhase.java:230)\r\n\tat org.elasticsearch.action.search.FetchSearchPhase.lambda$innerRun$1(FetchSearchPhase.java:118)\r\n\tat org.elasticsearch.action.search.CountedCollector.countDown(CountedCollector.java:51)\r\n\tat org.elasticsearch.action.search.CountedCollector.onFailure(CountedCollector.java:70)\r\n\tat org.elasticsearch.action.search.FetchSearchPhase$2.onFailure(FetchSearchPhase.java:197)\r\n\tat org.elasticsearch.action.ActionListenerResponseHandler.handleException(ActionListenerResponseHandler.java:59)\r\n\tat org.elasticsearch.action.search.SearchTransportService$ConnectionCountingHandler.handleException(SearchTransportService.java:403)\r\n\tat org.elasticsearch.transport.TransportService$6.handleException(TransportService.java:579)\r\n\tat org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.handleException(TransportService.java:1112)\r\n\tat org.elasticsearch.transport.TransportService$DirectResponseChannel.processException(TransportService.java:1221)\r\n\tat org.elasticsearch.transport.TransportService$DirectResponseChannel.sendResponse(TransportService.java:1195)\r\n\tat org.elasticsearch.transport.TaskTransportChannel.sendResponse(TaskTransportChannel.java:61)\r\n\tat org.elasticsearch.transport.TransportChannel.sendErrorResponse(TransportChannel.java:56)\r\n\tat org.elasticsearch.action.support.ChannelActionListener.onFailure(ChannelActionListener.java:51)\r\n\tat org.elasticsearch.action.ActionRunnable.onFailure(ActionRunnable.java:88)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:39)\r\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:44)\r\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:706)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\r\n\tat java.base/java.lang.Thread.run(Thread.java:832)\r\nCaused by: java.lang.IllegalArgumentException: Invalid format: [weekyear]: Unknown pattern letter: r\r\n\tat org.elasticsearch.common.time.DateFormatters.forPattern(DateFormatters.java:1815)\r\n\tat org.elasticsearch.common.time.DateFormatter.forPattern(DateFormatter.java:145)\r\n\tat org.elasticsearch.index.mapper.DateFieldMapper$DateFieldType.docValueFormat(DateFieldMapper.java:436)\r\n\tat org.elasticsearch.search.fetch.subphase.FetchDocValuesPhase.hitsExecute(FetchDocValuesPhase.java:87)\r\n\tat org.elasticsearch.search.fetch.FetchPhase.execute(FetchPhase.java:184)\r\n\tat org.elasticsearch.search.SearchService.lambda$executeFetchPhase$4(SearchService.java:585)\r\n\tat org.elasticsearch.action.ActionRunnable.lambda$supply(ActionRunnable.java:58)\r\n\tat org.elasticsearch.action.ActionRunnable$2.doRun(ActionRunnable.java:73)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)\r\n\t... 6 more\r\nCaused by: java.lang.IllegalArgumentException: Unknown pattern letter: r\r\n\tat java.base/java.time.format.DateTimeFormatterBuilder.parsePattern(DateTimeFormatterBuilder.java:1813)\r\n\tat java.base/java.time.format.DateTimeFormatterBuilder.appendPattern(DateTimeFormatterBuilder.java:1710)\r\n\tat org.elasticsearch.common.time.DateFormatters.forPattern(DateFormatters.java:1811)\r\n\t... 14 more\r\n"
    },
    "stack_trace" : "Failed to execute phase [fetch], all shards failed; shardFailures {[qIouw73JScmjqp8Y9eBJRw][project][0]: org.elasticsearch.transport.RemoteTransportException: [readonly-node-f48b3e9200][127.0.0.1:9300][indices:data/read/search[phase/fetch/id]]\r\nCaused by: java.lang.IllegalArgumentException: Invalid format: [weekyear]: Unknown pattern letter: r\r\n\tat org.elasticsearch.common.time.DateFormatters.forPattern(DateFormatters.java:1815)\r\n\tat org.elasticsearch.common.time.DateFormatter.forPattern(DateFormatter.java:145)\r\n\tat org.elasticsearch.index.mapper.DateFieldMapper$DateFieldType.docValueFormat(DateFieldMapper.java:436)\r\n\tat org.elasticsearch.search.fetch.subphase.FetchDocValuesPhase.hitsExecute(FetchDocValuesPhase.java:87)\r\n\tat org.elasticsearch.search.fetch.FetchPhase.execute(FetchPhase.java:184)\r\n\tat org.elasticsearch.search.SearchService.lambda$executeFetchPhase$4(SearchService.java:585)\r\n\tat org.elasticsearch.action.ActionRunnable.lambda$supply(ActionRunnable.java:58)\r\n\tat org.elasticsearch.action.ActionRunnable$2.doRun(ActionRunnable.java:73)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)\r\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:44)\r\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:706)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\r\n\tat java.base/java.lang.Thread.run(Thread.java:832)\r\nCaused by: java.lang.IllegalArgumentException: Unknown pattern letter: r\r\n\tat java.base/java.time.format.DateTimeFormatterBuilder.parsePattern(DateTimeFormatterBuilder.java:1813)\r\n\tat java.base/java.time.format.DateTimeFormatterBuilder.appendPattern(DateTimeFormatterBuilder.java:1710)\r\n\tat org.elasticsearch.common.time.DateFormatters.forPattern(DateFormatters.java:1811)\r\n\t... 14 more\r\n}{[qIouw73JScmjqp8Y9eBJRw][project][1]: org.elasticsearch.transport.RemoteTransportException: [readonly-node-f48b3e9200][127.0.0.1:9300][indices:data/read/search[phase/fetch/id]]\r\nCaused by: java.lang.IllegalArgumentException: Invalid format: [weekyear]: Unknown pattern letter: r\r\n\tat org.elasticsearch.common.time.DateFormatters.forPattern(DateFormatters.java:1815)\r\n\tat org.elasticsearch.common.time.DateFormatter.forPattern(DateFormatter.java:145)\r\n\tat org.elasticsearch.index.mapper.DateFieldMapper$DateFieldType.docValueFormat(DateFieldMapper.java:436)\r\n\tat org.elasticsearch.search.fetch.subphase.FetchDocValuesPhase.hitsExecute(FetchDocValuesPhase.java:87)\r\n\tat org.elasticsearch.search.fetch.FetchPhase.execute(FetchPhase.java:184)\r\n\tat org.elasticsearch.search.SearchService.lambda$executeFetchPhase$4(SearchService.java:585)\r\n\tat org.elasticsearch.action.ActionRunnable.lambda$supply(ActionRunnable.java:58)\r\n\tat org.elasticsearch.action.ActionRunnable$2.doRun(ActionRunnable.java:73)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)\r\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:44)\r\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:706)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\r\n\tat java.base/java.lang.Thread.run(Thread.java:832)\r\nCaused by: java.lang.IllegalArgumentException: Unknown pattern letter: r\r\n\tat java.base/java.time.format.DateTimeFormatterBuilder.parsePattern(DateTimeFormatterBuilder.java:1813)\r\n\tat java.base/java.time.format.DateTimeFormatterBuilder.appendPattern(DateTimeFormatterBuilder.java:1710)\r\n\tat org.elasticsearch.common.time.DateFormatters.forPattern(DateFormatters.java:1811)\r\n\t... 14 more\r\n}\r\n\tat org.elasticsearch.action.search.AbstractSearchAsyncAction.onPhaseFailure(AbstractSearchAsyncAction.java:554)\r\n\tat org.elasticsearch.action.search.AbstractSearchAsyncAction.executeNextPhase(AbstractSearchAsyncAction.java:309)\r\n\tat org.elasticsearch.action.search.FetchSearchPhase.moveToNextPhase(FetchSearchPhase.java:230)\r\n\tat org.elasticsearch.action.search.FetchSearchPhase.lambda$innerRun$1(FetchSearchPhase.java:118)\r\n\tat org.elasticsearch.action.search.CountedCollector.countDown(CountedCollector.java:51)\r\n\tat org.elasticsearch.action.search.CountedCollector.onFailure(CountedCollector.java:70)\r\n\tat org.elasticsearch.action.search.FetchSearchPhase$2.onFailure(FetchSearchPhase.java:197)\r\n\tat org.elasticsearch.action.ActionListenerResponseHandler.handleException(ActionListenerResponseHandler.java:59)\r\n\tat org.elasticsearch.action.search.SearchTransportService$ConnectionCountingHandler.handleException(SearchTransportService.java:403)\r\n\tat org.elasticsearch.transport.TransportService$6.handleException(TransportService.java:579)\r\n\tat org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.handleException(TransportService.java:1112)\r\n\tat org.elasticsearch.transport.TransportService$DirectResponseChannel.processException(TransportService.java:1221)\r\n\tat org.elasticsearch.transport.TransportService$DirectResponseChannel.sendResponse(TransportService.java:1195)\r\n\tat org.elasticsearch.transport.TaskTransportChannel.sendResponse(TaskTransportChannel.java:61)\r\n\tat org.elasticsearch.transport.TransportChannel.sendErrorResponse(TransportChannel.java:56)\r\n\tat org.elasticsearch.action.support.ChannelActionListener.onFailure(ChannelActionListener.java:51)\r\n\tat org.elasticsearch.action.ActionRunnable.onFailure(ActionRunnable.java:88)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:39)\r\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:44)\r\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:706)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\r\n\tat java.base/java.lang.Thread.run(Thread.java:832)\r\nCaused by: org.elasticsearch.ElasticsearchException$1: Invalid format: [weekyear]: Unknown pattern letter: r\r\n\tat org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:644)\r\n\tat org.elasticsearch.action.search.AbstractSearchAsyncAction.executeNextPhase(AbstractSearchAsyncAction.java:307)\r\n\t... 22 more\r\nCaused by: java.lang.IllegalArgumentException: Invalid format: [weekyear]: Unknown pattern letter: r\r\n\tat org.elasticsearch.common.time.DateFormatters.forPattern(DateFormatters.java:1815)\r\n\tat org.elasticsearch.common.time.DateFormatter.forPattern(DateFormatter.java:145)\r\n\tat org.elasticsearch.index.mapper.DateFieldMapper$DateFieldType.docValueFormat(DateFieldMapper.java:436)\r\n\tat org.elasticsearch.search.fetch.subphase.FetchDocValuesPhase.hitsExecute(FetchDocValuesPhase.java:87)\r\n\tat org.elasticsearch.search.fetch.FetchPhase.execute(FetchPhase.java:184)\r\n\tat org.elasticsearch.search.SearchService.lambda$executeFetchPhase$4(SearchService.java:585)\r\n\tat org.elasticsearch.action.ActionRunnable.lambda$supply(ActionRunnable.java:58)\r\n\tat org.elasticsearch.action.ActionRunnable$2.doRun(ActionRunnable.java:73)\r\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)\r\n\t... 6 more\r\nCaused by: java.lang.IllegalArgumentException: Unknown pattern letter: r\r\n\tat java.base/java.time.format.DateTimeFormatterBuilder.parsePattern(DateTimeFormatterBuilder.java:1813)\r\n\tat java.base/java.time.format.DateTimeFormatterBuilder.appendPattern(DateTimeFormatterBuilder.java:1710)\r\n\tat org.elasticsearch.common.time.DateFormatters.forPattern(DateFormatters.java:1811)\r\n\t... 14 more\r\n"
  },
  "status" : 400
}
```

--- a/tests/Tests/Search/Search/SearchApiTests.cs
+++ b/tests/Tests/Search/Search/SearchApiTests.cs
@@ -285,7 +285,7 @@ namespace Tests.Search.Search
 				new
 				{
 					field = "lastActivity",
-					format = "weekyear"
+					format = DateFormat.basic_date
 				},
 			}
 		};
@@ -306,7 +306,7 @@ namespace Tests.Search.Search
 			)
 			.DocValueFields(fs => fs
 				.Field(p => p.Name)
-				.Field(p => p.LastActivity, format: "weekyear")
+				.Field(p => p.LastActivity, format: DateFormat.basic_date)
 			);
 
 		protected override SearchRequest<Project> Initializer => new SearchRequest<Project>()
@@ -324,7 +324,7 @@ namespace Tests.Search.Search
 				Value = "Stable"
 			}),
 			DocValueFields = Infer.Field<Project>(p => p.Name)
-				.And<Project>(p => p.LastActivity, format: "weekyear")
+				.And<Project>(p => p.LastActivity, format: DateFormat.basic_date)
 		};
 
 		protected override void ExpectResponse(ISearchResponse<Project> response)

